libres/old_tests/job_queue/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+# -----------------------------------------------------------------
+# Additional dependencies
+# -----------------------------------------------------------------
```

Comment:
Is this file new? If not then the diff seems to not add a mv here because the file is too different, might want to do some doctoring of the commits to make that happen.

If it's new, some comment in a PR or commit about why would be nice